aioredisqueue/queue.py

```diff
@@ -14,6 +15,9 @@ def __init__(self, redis, key_prefix='aioredisqueue_', *,
                  lua_sha=None,
                  loop=None):
 
+        if key_prefix is None:
+            key_prefix = 'aioredisqueue_' + serialization_method
```

Comment:
the cases of `serialization_method` being  None (for `raw`) or `dynamic`  should be handled and documented. I'm not sure if we need `serialization_method` in queue key if we'll have it in task_ids.

Reply:
The reason for having it as `queue` constructor parameter is it just provides a way to specify the default encoder different from `json` once instead of every time when putting a new task. Yes, having it at the same time as Redis key is actually data duplication, but it provides the easiest usage for the very popular cases of using json or custom loader just by specifying it once in each producer

Reply:
Yeah, maybe, but if I specify `serialization_method='raw'` or (which is meant to be the same) `serialization_method=None`,  `key_prefix` shouldn't end with `_raw` and with `None` you'll have an error here.